src/coreComponents/linearAlgebra/interfaces/hypre/HypreInterface.cpp

```diff
@@ -44,11 +44,11 @@ void HypreInterface::initialize()
   HYPRE_SetMemoryLocation( hypre::memoryLocation );
 
   // Hypre version info
-#if defined(HYPRE_DEVELOP_STRING) && defined(HYPRE_BRANCH_NAME)
-  GEOSX_LOG_RANK_0( "  Using hypre development version: " << HYPRE_DEVELOP_STRING <<
+#if defined(HYPRE_DEVELOP_STRING) 
```

Comment:
Should this be moved to the newly added `outputVersionInfo` such that all the version stuff can be found in a single place?

Reply:
Also, I think it's better to leave the check `defined(HYPRE_BRANCH_NAME)` in there because this variable has been added this week while `HYPRE_DEVELOP_STRING` exists for nearly a year. If someone tries to use a commit in hypre in that window (last year - this week), this would break. This is very unlikely, but just to be in the same side...

Reply:
> Also, I think it's better to leave the check `defined(HYPRE_BRANCH_NAME)` in there because this variable has been added this week while `HYPRE_DEVELOP_STRING` exists for nearly a year. If someone tries to use a commit in hypre in that window (last year - this week), this would break. This is very unlikely, but just to be in the safe side...

Yeah...I had something more elaborate here....I don't know what happened....I'll fix it again.